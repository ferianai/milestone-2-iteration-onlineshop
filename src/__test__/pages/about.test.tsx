import { render, screen } from '@testing-library/react';
import AboutPage from '@/pages/about';

describe('AboutPage', () => {
  it('renders the About ShopSmart Page', () => {
    const page = render(<AboutPage />);
    expect(page).toMatchSnapshot();
  });
});
