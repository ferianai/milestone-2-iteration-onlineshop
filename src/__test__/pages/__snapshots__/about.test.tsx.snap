// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage renders the About Page 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1>
        About Page
      </h1>
    </div>
  </body>,
  "container": <div>
    <h1>
      About Page
    </h1>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`AboutPage renders the About ShopSmart Page 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="w-full min-h-screen flex flex-col items-center"
      >
        <header
          class="w-full bg-gray-100 text-black py-8 text-center text-6xl font-bold"
        >
          About ShopSmart
        </header>
        <section
          class="w-full min-h-screen flex flex-col justify-center items-center bg-gray-100 p-8"
        >
          <h2
            class="text-4xl font-semibold mb-4"
          >
            Our Vision & Mission
          </h2>
          <p
            class="text-lg text-center max-w-4xl"
          >
            At ShopSmart, our vision is to revolutionize online shopping by providing top-quality products at unbeatable prices. Our mission is to create a seamless and enjoyable shopping experience, ensuring customer satisfaction with every purchase.
          </p>
        </section>
        <section
          class="w-full min-h-screen flex flex-col justify-center items-center bg-white p-8"
        >
          <h2
            class="text-4xl font-semibold mb-4"
          >
            Meet Our Team
          </h2>
          <p
            class="text-lg text-center max-w-4xl"
          >
            Our dedicated team at ShopSmart is committed to delivering the best shopping experience. From product sourcing to customer support, we work together to ensure excellence.
          </p>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="w-full min-h-screen flex flex-col items-center"
    >
      <header
        class="w-full bg-gray-100 text-black py-8 text-center text-6xl font-bold"
      >
        About ShopSmart
      </header>
      <section
        class="w-full min-h-screen flex flex-col justify-center items-center bg-gray-100 p-8"
      >
        <h2
          class="text-4xl font-semibold mb-4"
        >
          Our Vision & Mission
        </h2>
        <p
          class="text-lg text-center max-w-4xl"
        >
          At ShopSmart, our vision is to revolutionize online shopping by providing top-quality products at unbeatable prices. Our mission is to create a seamless and enjoyable shopping experience, ensuring customer satisfaction with every purchase.
        </p>
      </section>
      <section
        class="w-full min-h-screen flex flex-col justify-center items-center bg-white p-8"
      >
        <h2
          class="text-4xl font-semibold mb-4"
        >
          Meet Our Team
        </h2>
        <p
          class="text-lg text-center max-w-4xl"
        >
          Our dedicated team at ShopSmart is committed to delivering the best shopping experience. From product sourcing to customer support, we work together to ensure excellence.
        </p>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
